scss/tech.scss

```diff
@@ -18,9 +18,10 @@
   background-color: $earls-green;
   display: flex;
   flex-direction: column;
-  left: 50px;
+  left: 45px;
+  margin: 10px 15px 10px 15px;
```

Comment:
There's another shorthand for this, using only two params, horizontal and vertical